src/components/Feed.js

```diff
@@ -5,62 +5,136 @@ import { axiosInstance } from '../util/config';
-                    <div className="form-group form-element">
-                        <input name="subject" type="text" className="form-control" placeholder="Title" value={updateInput.subject} onChange={handleInputChange} />
-                    </div>
-                    <div className="form-group  form-element">
-                        <input name="body" type="textarea" className="form-control" placeholder="Updates" value={updateInput.body} onChange={handleInputChange} />
-                    </div>
-                    <button className="btn btn-primary form-element" onClick={handleAddUpdate}>Post</button>
-                </div>
-            </div>
+    axiosInstance
+      .post(
+        `/projects/${updateInput.project}/updates`,
+        { subject, body },
+        { headers: { authorization: 'Bearer ' + authorization } }
+      )
+      .then((res) => {
+        console.log('res-post:', res);
+      });
+  };
+
+  const getProjectsList = () => {
+    axiosInstance.get(`/users/${username}`).then((res) => {
+      setProjectData(res.data.projectMemberships);
+      setUpdateInput({
+        ...updateInput,
+        project: res.data.projectMemberships[0].projectId,
+      });
+    });
+  };
+
+  useEffect(() => {
+    getProjectsList();
+  }, []);
+
+  return (
+    <div className='postapp'>
+      <div className='page-title'>
+        <h1>Feed</h1>
+      </div>
+      <div className='project-form'>
+        <div className='form-group form-element'>
+          <select
+            name='project'
+            onChange={handleInputChange}
+            className='form-select'
+          >
+            {projectData?.map((project, index) => (
+              <option key={`project-option-${index}`} value={project.projectId}>
+                {project.projectName}
+              </option>
+            ))}
+          </select>
         </div>
-    );
-}
\ No newline at end of file
+        <div className='form-group form-element'>
+          <input
+            name='subject'
+            type='text'
+            className='form-control'
+            placeholder='Title'
+            value={updateInput.subject}
+            onChange={handleInputChange}
+          />
+        </div>
+        <div className='form-group  form-element'>
+          <input
+            name='body'
+            type='textarea'
+            className='form-control'
+            placeholder='Updates'
+            value={updateInput.body}
+            onChange={handleInputChange}
+          />
+        </div>
+        <button
+          className='btn btn-primary form-element'
+          onClick={handleAddUpdate}
+        >
+          Post
+        </button>
+      </div>
+      <div className='project-updates'>
+        {updatesList?.map((update, index) => (
+          <div key={`project-update-item-${index}`} className='update-card'>
+            <div className='custom-card-header'>
+              <h2>{update.projectName}</h2>
+              <h4>{update.author}</h4>
```

Comment:
Header tags are typically used in order (from `h1` to `h2` to `h3` to ...). If you want different sizes, it's best to use an `h3` here and update the styling to reflect your desired size.